types/Foundation/Configuration/Middleware.php

```diff
@@ -21,9 +21,9 @@
     '192.168.1.2',
 ]);
 
-$middleware->trustProxies(at: '*', withHeaders: Request::HEADER_X_FORWARDED_AWS_ELB);
+$middleware->trustProxies(at: '*', headers: Request::HEADER_X_FORWARDED_AWS_ELB);
```

Comment:
I didn't modify the TrustProxies middleware itself, but it seems that someone changed the parameter name and didn't update the test. I made this adjustment to satisfy the CI type checker.

Reply:
We should just not use named parameters tbh. Most libraries don't make them part of their BC promise.